plugins/easyimage/plugin.js

```diff
@@ -453,4 +556,80 @@
 	 * @member CKEDITOR.config
 	 */
 	CKEDITOR.config.easyimage_sideClass = 'easyimage-side';
+
+	/**
+	 * A CSS class representing image aligned to left.
+	 *
+	 *		// Changes the class to "my-left-image".
+	 *		config.easyimage_alignLeftClass = 'my-left-image';
+	 *
+	 * @since 4.9.0
+	 * @cfg {String} [easyimage_alignLeftClass='easyimage-alignLeft']
+	 * @member CKEDITOR.config
+	 */
+	CKEDITOR.config.easyimage_alignLeftClass = 'easyimage-alignLeft';
+
+	/**
+	 * A CSS class representing image aligned to center.
+	 *
+	 *		// Changes the class to "my-center-image".
+	 *		config.easyimage_alignCenterClass = 'my-center-image';
+	 *
+	 * @since 4.9.0
+	 * @cfg {String} [easyimage_alignCenterClass='easyimage-alignCenter']
+	 * @member CKEDITOR.config
+	 */
+	CKEDITOR.config.easyimage_alignCenterClass = 'easyimage-alignCenter';
+
+	/**
+	 * A CSS class representing image aligned to right.
+	 *
+	 *		// Changes the class to "my-right-image".
+	 *		config.easyimage_alignLeftClass = 'my-right-image';
+	 *
+	 * @since 4.9.0
+	 * @cfg {String} [easyimage_alignRightClass='easyimage-alignRight']
+	 * @member CKEDITOR.config
+	 */
+	CKEDITOR.config.easyimage_alignRightClass = 'easyimage-alignRight';
+
+	/**
+	 * Custom styles that could be applied to Easy Image widget. All styles must be instances of
+	 * {@link CKEDITOR.style}. There are three additional properties for every style:
+	 *
+	 * * `label` - string used as a button label in a balloon toolbar for the widget,
+	 * * `icon` - path to the icon used in the balloon toolbar,
+	 * * `iconHiDpi` - path to the high DPI version of the icon.
+	 *
+	 *		config.easyimage_styles = {
```

Comment:
Code listing is not displayed properly:

![image](https://user-images.githubusercontent.com/5353898/35633542-cf59c7fc-06a9-11e8-878c-7361498ec76a.png)

Reply:
Reported as ckeditor/jsduck#8. I've mitigated it by moving note "Every style added […]" directly under the list.